### PR TITLE
Screenings: Add No Man's Land

### PR DESCRIPTION
**Pull request by mglidden**

### CODE REVIEW

File: _pages/events/screenings.md [hunk 1]
```diff
@@ -7,6 +7,40 @@
...
 
+### No Man’s Land Film Festival
+
+on **Thursday, February 27 @ 7:00 PM** in [MIT Building 26-100](http://whereis.mit.edu/?go=26)
+
+**_$5 with a valid MIT ID or $10 for General Admission_**
+
+**Purchase your ticket today online!**
+
+Click [here](/pay) or go to the Pay MITOC link above
```
Comment (DavidCain) [inline _pages/events/screenings.md]: Generally best to [avoid saying click here](https://www.w3.org/QA/Tips/noClickHere) (for accessibility reasons). Instead, maybe just `[Pay MITOC](/pay) and under Trip and Event Fees,...`

File: _pages/events/screenings.md [hunk 2]
```diff
@@ -7,6 +7,40 @@
...
+With the help of theatres, nonprofits, climbing gyms and other bold hosts and venues across the globe, No Man's Land is sharing its mission of un-defining feminine and what it means to be a woman. While the name may sound exclusive, No Man’s Land encourages all genders to attend. Beyond the No Man's Land's mission, the films are a strong curation and representation of true outdoor adventure film.
```
Comment (DavidCain) [inline _pages/events/screenings.md]: Add a newline beforehand to make this a new paragraph?

File: _pages/events/screenings.md [hunk 3]
```diff
@@ -7,6 +7,40 @@
...
+With the help of theatres, nonprofits, climbing gyms and other bold hosts and venues across the globe, No Man's Land is sharing its mission of un-defining feminine and what it means to be a woman. While the name may sound exclusive, No Man’s Land encourages all genders to attend. Beyond the No Man's Land's mission, the films are a strong curation and representation of true outdoor adventure film.			
+For more information, please visit [www.nomanslandfilmfestival.org](www.nomanslandfilmfestival.org).
```
Comment (DavidCain) [inline _pages/events/screenings.md]: Should use `https://` because, sadly, the site doesn't redirect from HTTP.

**Comment by fpagnoux** [comment]
In markdown, 2 trailing spaces mean a line break (without changing paragraph), so generally you don't want to let your editor remove these (not a fan of this convention, but this is what it is).

I'll restore these and adjust the image size

**Comment by DavidCain** [comment]
Oh, yikes. All apologies - I almost always edit Markdown with trailing whitespace automatically stripped, so I forget that that this quirk of Markdown exists.

Should we use a `<br>` instead to be more explicit about what this trailing whitespace is meant to do?

(And thanks for fixing the image size - I only noticed that once deployed).

**Comment by fpagnoux** [comment]
> Should we use a <br> instead to be more explicit about what this trailing whitespace is meant to do?

That sounds like a good idea 🙂

**Pull request merged**